src/rgw/rgw_rest_role.cc

```diff
@@ -35,8 +35,8 @@ int RGWRestRole::verify_permission(optional_yield y)
     return op_ret;
   }
 
+  _role = std::move(role);
```

Comment:
Does this change allow system users to continue issue role policy requests, even when system users fail the permission check?